docs/driver_installation_zadig.md

```diff
@@ -52,15 +52,15 @@ Find the `Inf name` value in the Details tab of the device properties. This shou
 
 ![Device properties showing the Inf name value](https://i.imgur.com/Bu4mk9m.png)
 
-Then, open a new Command Prompt window as an Administrator (type in `cmd` into the Start menu and press Ctrl+Shift+Enter). Run `pnputil /enum-drivers` to verify the `Inf name` matches the `Published Name` field of one of the entries:
+Then, open a new Command Prompt window as an Administrator (type in `cmd` into the Start menu and press Ctrl+Shift+Enter). Run `pnputil /enum-drivers`(`pnputil -e` for Windows 7 and later) to verify the `Inf name` matches the `Published Name` field of one of the entries:
```

Comment:
```suggestion
Then, open a new Command Prompt window as an Administrator (type in `cmd` into the Start menu and press Ctrl+Shift+Enter). Run `pnputil /enum-drivers` to verify the `Inf name` matches the `Published Name` field of one of the entries:
```
The long form still works on Win 7 and up, and is a little clearer as to what it's doing.